opt-out.md

```diff
@@ -7,13 +7,20 @@ To determine if Allstar is configured in the opt-out strategy, there will be a r
     optConfig:
       optOutStrategy: true
 
-To opt-out, submit a PR to that `.allstar` repo, and add the name of your repository to the opt-out list. ex:
+To opt-out specific repository, submit a PR to that `.allstar` repo, and add the name of your repository to the opt-out list. ex:
 
     optConfig:
       optOutStrategy: true
       optOutRepos:
       - my-repo-name-here
 
+To opt-out all private/public repositories, submit a PR to that `.allstar` repo, and add `optOutPrivateRepos` or `optOutPublicRepos`. ex:
```

Comment:
I'd like to keep this file focused on a single repository opt-out. Please revert this, and add this section to the end of the section in https://github.com/ossf/allstar/blob/main/README.md#enable-configuration

Reply:
Thanks. Updated with 48d36c7d664c0d821e438e19a94a511db9f054c6 and 75eee18e91963f2727285dbd0ab32ee42c4503e0